examples/complete-http/main.tf

```diff
@@ -12,8 +12,7 @@ provider "aws" {
 }
 
 locals {
-  domain_name = "terraform-aws-modules.modules.tf" # trimsuffix(data.aws_route53_zone.this.name, ".")
```

Comment:
This is a real subdomain I use for tests, let's use it here. We can discuss how to share a zone later when we have to run these examples in automation.

Also, we don't have to ask users to edit the content of `terraform.tfvars` in order to run examples. They always go with `terraform init,plan,apply,destroy`.

Reply:
added as the default value for the variable to maintain current functionality but also so others (myself included) can test with a domain